pose-detection/src/calculators/get_object_scale.ts

```diff
@@ -14,19 +14,17 @@
  * limitations under the License.
  * =============================================================================
  */
-import {ImageSize} from './interfaces/common_interfaces';
 import {Rect} from './interfaces/shape_interfaces';
 
 /**
  * Estimate object scale to allow filter work similarly on nearer or futher
  * objects.
- * @param roi Normalized rectangle.
- * @param imageSize An object that contains width and height.
+ * @param roi Non-normalized rectangle.
  * @returns A number representing the object scale.
  */
-export function getObjectScale(roi: Rect, imageSize: ImageSize): number {
```

Comment:
This commit doesn't look like support of non-normalized keypoints.
Do you just remove scaling in the file here?
If so - I don't see it reflected in the code that actually calls this and other methods.

Reply:
This change was temporary and is now gone in the newer commits